### ISSUE TITLE
hax-preview keyboard traps incorrectly

### ISSUE THREAD

**Issue by mgifford** [LRNWebComponents/hax-body#128]
Keyboard only users cannot navigate to make pages like:

<img width="1024" alt="Make page example" src="https://user-images.githubusercontent.com/116832/41055934-94aafe88-6990-11e8-9b5a-d65d4c5457ab.png">

There is a keyboard trap which stops the user from navigating into add the content on the form.  It works when you are in there, but...

**Comment by btopro** [comment]
odd; seems to be a keyboard trap that sticks to cancel; clicking past it then its kept in the rest of the form. Will have to dig more on this as the split layout tag in its current implementation appears to be throttling this for some reason

**Comment by btopro** [comment]
fixed this. related to #131 which that will need a more prevasive solution for the other drawer tags but the solution in this one will forge the path to those
